helm/charts/hydra/charts/hydra-maester/values.yaml

```diff
@@ -14,6 +14,10 @@ image:
   # Image pull policy
   pullPolicy: IfNotPresent
 
+adminService:
```

Comment:
I believe there is no need to define additional keys for this, see: https://v3.helm.sh/docs/topics/chart_template_guide/subcharts_and_globals/#overriding-values-from-a-parent-chart

Reply:
Oh, I didn't know that - I thought the parent chart is the "global" value per default. Thank you for clarifying that!

An alternative would be to use a template for this, as templates can be referenced directly from sub-charts: https://helm.sh/docs/chart_template_guide/#sharing-templates-with-subcharts

That way, there would be no extra configuration required for the maester chart, it would just work if enabled - which would be super awesome!

Reply:
The issue with sharing templates is that you still have to pass values into the template (the `.` in `{{ include "hydra-masetr.adminService.name" . }}`). The template will still render but it will use the values passed in. Since we don't have access to the parent chart values, we can't pass them into the templates either, so calling `{{ include "hydra.fullname" . }}` from the sub chart would use the `fullnameOverride` from the sub chart values, not the parent chart.

As far as I can tell from my research, the only options are to either have the keys defined in the values or to combine the two applications into a single chart rather than a parent/sub chart.

Reply:
Damn! Ok, thank you for being so thorough. I think combining charts could be a solution, I think the less we have to configure to get things working, the better for the end user - as long as all functionality is still accessible.

I can imagine a case where a developer changes the port to e.g. 80 and forgets to set up the other services correctly - it will take some time to debug this.

What do you think about merging the charts?

Reply:
I think that is probably the best solution, as you'd likely never need to deploy the maester without the main chart. I can work this up and I'll update the PR

Reply:
I am going to be deploying oathkeeper as well soon, should I give that chart the same treatment?

Reply:
Awesome - thank you paul! Yes that would be very useful :)

Reply:
It looks like the CRD for the version of the maester running is outdated for the latest version of the maester (which is tagged for oryOS11, but the 1.0.0 for hydra is oryOS12)

Should the versions in the charts be updated to the latest versions as a part of this?

Reply:
yes!